libmscore/noteentry.cpp

```diff
@@ -340,6 +340,15 @@ void Score::putNote(const Position& p, bool replace)
       ChordRest* cr = _is.cr();
       bool addToChord = false;
 
+      if(_graceInputState != IconType::NONE && selection().elements().count() == 1){
+            if (cr && cr->isChord()) {
+                        if(selection().element()->type() == ElementType::NOTE){
+                        setGraceNote(toChord(cr), nval.pitch, iconTypeToNoteType(_graceInputState), iconTypeToLen(_graceInputState));
+                        score()->addRefresh(cr->canvasBoundingRect());
+                        }
+                  }
+            return;
+            }
```

Comment:
Please see https://musescore.org/en/handbook/developers-handbook/finding-your-way-around/musescore-coding-rules#Coding-style and make sure to maintain the spacing rules exactly.  BTW, if you use QtCreator, you can download that qt2-musescore.xml and import it into your QtCreator to automatically use those rules.

Reply:
I fixed it and commited using --fixup & --autosquash as recommended by "Git worklow" of MuseScore. Seems no good idea. My local git tells me I am up to date, but remote system says I am behind.
Dou you have any hint for me?